app/code/Magento/CustomerGraphQl/Model/Customer/ValidateCustomerData.php

```diff
@@ -32,25 +41,40 @@ class ValidateCustomerData
      *
      * @param GetAllowedCustomerAttributes $getAllowedCustomerAttributes
      * @param EmailAddressValidator $emailAddressValidator
+     * @param CustomerMetadataInterface $customerMetadata
      */
     public function __construct(
         GetAllowedCustomerAttributes $getAllowedCustomerAttributes,
-        EmailAddressValidator $emailAddressValidator
+        EmailAddressValidator $emailAddressValidator,
+        CustomerMetadataInterface $customerMetadata
```

Comment:
Please make sure to validate constructor params for null, when adding a new param to an existing class. something in the lines of  $customerMetadata ?: \Magento\Framework\App\ObjectManager::getInstance()
            ->get(CustomerMetadataInterface::class);

Reply:
This comment is outdate now.